mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/repositories/EnhetRepository.kt

```diff
@@ -0,0 +1,49 @@
+package no.nav.mulighetsrommet.api.repositories
+
+import kotliquery.Row
+import kotliquery.queryOf
+import no.nav.mulighetsrommet.api.domain.dbo.EnhetStatus
+import no.nav.mulighetsrommet.api.domain.dbo.Norg2EnhetDbo
+import no.nav.mulighetsrommet.database.Database
+import no.nav.mulighetsrommet.database.utils.QueryResult
+import no.nav.mulighetsrommet.database.utils.query
+import org.intellij.lang.annotations.Language
+import org.slf4j.LoggerFactory
+
+class EnhetRepository(private val db: Database) {
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    fun upsert(enhet: Norg2EnhetDbo): QueryResult<Norg2EnhetDbo> = query {
+        logger.info("Lagrer enhet id=${enhet.enhetId}")
+
+        @Language("PostgreSQL")
+        val query = """
+            insert into enhet(enhetid, navn, enhetsnummer, status)
```

Comment:
Har kanskje ikke så mye å si, men ser at casing på `enhetid` og  `enhetId` i sql er forskjellig. Vi kunne også navngitt denne kolonnen for `enhet_id` for å følge samme konvensjoner som vi har gjort i andre databasetabeller